pkg/kubectl/cmd/run.go

```diff
@@ -562,7 +623,11 @@ func generateService(f cmdutil.Factory, cmd *cobra.Command, args []string, servi
 	return nil
 }
 
-func createGeneratedObject(f cmdutil.Factory, cmd *cobra.Command, generator kubectl.Generator, names []kubectl.GeneratorParam, params map[string]interface{}, overrides, namespace string) (runtime.Object, string, meta.RESTMapper, *meta.RESTMapping, error) {
+func createGeneratedObject(f cmdutil.Factory, cmd *cobra.Command,
```

Comment:
> do we have a rule for that ?

No, we don't have to follow any strict line length rules.
If I am working in a function, I tend to break the lines in such a way that two files can be viewed side-by-side. It's not more correct or less correct; the style guide says it doesn't matter. I can change it back if you think that would be better.